tools/code_format/check_format.py

```diff
@@ -755,10 +755,10 @@ def checkSourceLine(line, file_path, reportError):
                     "Grpc::GoogleGrpcContext. See #8282")
 
   if denylistedForExceptions(file_path):
-    throw = line.find("throw")
-    if throw != -1:
+    # changed to accommodate cases where throw is a substring of a token like in "foothrowBar"
```

Comment:
nit: capitalize and re-word ("changed" doesn't mean anything to a viewer now) 

Seems like this change now correctly skips cases where throw is a substring of a token? If so maybe make the comment like `# Skip cases where throw is...`

Reply:
sure